src/js/App/Sidenav/Navigation.js

```diff
@@ -101,6 +102,7 @@ class Navigation extends Component {
                                                     && window.location.pathname.indexOf('/beta') === -1)) {
                                                     return <NavigationItem
                                                         itemID={subItem.reload || subItem.id}
+                                                        ouia-nav-item={item.id}
```

Comment:
I think that this should be ` ouia-nav-item={subItem.reload || subItem.id`}` otherwise this will be the same as ` ouia-nav-group` attr.

Reply:
Fixed!